src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -0,0 +1,253 @@
+        </span>
+      </div>
+    </div>
+
+    <!-- List Display Section -->
+    <div id="people-roles-list">
+      <!-- List Headers -->
+      <v-row class="people-roles-header list-item__subtitle" no-gutters>
+        <v-col v-for="(title, index) in tableHeaders" :key="index">
+          <span>{{title}}</span>
+        </v-col>
+        <!-- Spacer Column For Actions -->
+        <v-col sm="1" v-if="!isSummary"></v-col>
+      </v-row>
+
+      <!-- List Content -->
+      <v-row
+        class="people-roles-content"
+        :class="{ 'list-item__subtitle': !isSummary }"
+        v-for="(person, index) in personList"
+        :key="index"
+        no-gutters>
+        <v-col class="text-truncate">
+          <v-tooltip top :disabled="formatName(person).length < 25" color="primary">
+            <template v-slot:activator="{ on }">
+              <span v-on="on" class="people-roles-title"><strong>{{formatName(person)}}</strong></span>
+            </template>
+            <span>{{formatName(person)}}</span>
+          </v-tooltip>
+        </v-col>
+        <v-col>
+          <base-address class="peoples-roles-mailing-address" :address="person.address.mailingAddress" />
+        </v-col>
+        <v-col>
+          <p v-if="isSame(person.address.mailingAddress, person.address.deliveryAddress)"
+            class="peoples-roles-delivery-address">Same as Mailing Address
+          </p>
+          <base-address v-else class="peoples-roles-delivery-address" :address="person.address.deliveryAddress"/>
+        </v-col>
+        <v-col>
+          <div v-if="person.roles.length>0">
+            <v-col v-for="(role, index) in person.roles" :key="index" class="col-roles">
+              <span>{{role}}</span>
+            </v-col>
+          </div>
+          <div v-else>
+             <v-icon color="rgb(232, 156, 48)" small>mdi-alert</v-icon>
+             <span class="warningText">Add Role</span>
+          </div>
+        </v-col>
+
+        <!-- Actions Column -->
+        <v-col sm="1" v-if="!isSummary">
+          <div class="actions">
+            <span class="edit-action">
+              <v-btn small text color="primary"
+                :id="'person-' + person.id + '-change-btn'"
+                @click="emitPersonInfo(index)"
+              >
+                <v-icon small>mdi-pencil</v-icon>
+                <span>Edit</span>
+              </v-btn>
+            </span>
+
+            <!-- more actions menu -->
+            <span>
+              <v-menu offset-y>
+                <template v-slot:activator="{ on }">
+                  <v-btn text small color="primary" class="actions__more-actions__btn" v-on="on">
+                    <v-icon>mdi-menu-down</v-icon>
+                  </v-btn>
+                </template>
+                <v-list class="actions__more-actions">
+                  <v-list-item @click="confirmRemove(index)">
+                    <v-list-item-title><v-icon>mdi-delete</v-icon>Remove</v-list-item-title>
```

Comment:
This item looks too big, has too much vertical padding, and should be better aligned vertically.

Reply:
Fixed!